lib/node_modules/@stdlib/_tools/changelog/generate/lib/breaking_changes.js

```diff
@@ -54,9 +87,13 @@ var STDLIB_GITHUB_URL = 'https://github.com/stdlib-js/stdlib/commit';
 * // returns '-   [`abcdef1`](https://github.com/stdlib-js/stdlib/commit/abcdef1234567890): beep'
 */
 function formatBreakingChange( note ) {
-	var parts = note.text.split( '\n' );
-	var hash = trim( note.hash );
-	var out = '-   [`'+hash.substring( 0, 7 )+'`]('+STDLIB_GITHUB_URL+'/'+hash+'): '+parts[ 0 ];
+	var parts;
+	var hash;
+	var out;
+
+	parts = stripYamlBlocks( note.text ).split( '\n' );
```

Comment:
Seems unnecessary to split, join, split. Why can you not provide a list of lines to `stripYamlBlocks`, return a list, and then perform a trim via a slice by finding the first non-empty string index and the last non-empty string index?

Reply:
Also, trimming seems like something which is outside the scope of stripping YAML blocks. This is no longer relevant provided you refactor to use a slice approach which could happen in `formatBreakingChange`.